src/Datadog.Trace/Tracer.cs

```diff
@@ -344,6 +336,20 @@ public Span StartSpan(string operationName, ISpanContext parent = null, string s
                 }
             }
 
+            // automatically add the "env" tag if defined, taking precedence over an "env" tag set from a global tag
+            var env = Settings.Environment;
+            if (!string.IsNullOrWhiteSpace(env))
+            {
+                span.SetTag(Tags.Env, env);
+            }
+
+            // automatically add the "version" tag if defined, taking precdence over an "env" tag set from a global tag
```

Comment:
```suggestion
            // automatically add the "version" tag if defined, taking precdence over an "version" tag set from a global tag
```

Reply:
Fixed in c1208a9ef8269b501150bda94dea2b9823d8dbab